.travis.yml

```diff
@@ -13,18 +13,15 @@ env:
   matrix:
     - PGVERSION=9.5
     - PGVERSION=9.6
+    - PGVERSION=10
 before_install:
-  - git clone -b v0.6.1 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b pg10_hack --depth 1 https://github.com/citusdata/tools.git
```

Comment:
I opened citusdata/tools#17 to show what's changed.

Basically if those changes look good, I'll merge and release `0.7` or something and set Citus to use that in `master` (as `9.5` and `9.6` behavior is unaffected).